smartBugs.py

```diff
@@ -40,7 +40,7 @@ def analyse(args):
         sys.stdout.write('\x1b[1;34m' + file + '\x1b[0m')
         sys.stdout.write('\x1b[1;37m' + ' [' + tool + ']' + '\x1b[0m' + '\n')
 
-        analyse_files(tool, file, logs, output_folder, sarif_outputs, output_version, import_path)
+        analyse_files(tool, file, file_path_in_repo, logs, output_folder, sarif_outputs, output_version, import_path)
```

Comment:
Once the code is updated to use the NamedTuple, you can just pass it here. This is a bad smell... too many arguments.